### DISCUSSION TITLE
Editor integrations

### DISCUSSION THREAD

**Discussion by jack-michaud** [ast-grep/ast-grep#87] [Ideas]
Currently, only nvim is on the integrations page. Are there other integrations on the horizon? 

On my team, there is only one other person who uses vim. The others use JetBrains IDEs and VScode.

**Comment by HerringtonDarkholme** [comment]
Sure! Vscode support is on my roadmap!
Currently I focus more on the core lib and rule composition. 

I can make one simple vscode plugin soon. But it may be rustic.

**Comment by HerringtonDarkholme** [comment]
I have started a new repo to track the progress of vscode extension. https://github.com/ast-grep/ast-grep-vscode

**Reply by HerringtonDarkholme** [reply]
Current Status: a skeleton project

**Comment by HerringtonDarkholme** [comment]
A preview version is released on VSCode marketplace. https://marketplace.visualstudio.com/items?itemName=ast-grep.ast-grep-vscode&ssr=false#overview
